### DISCUSSION TITLE
[HW3] Is it possible to submit same paper with co-author?

### DISCUSSION THREAD

**Discussion by cpark88** [seominjoon/kaist-ai605#50] [Q&A] [answered]
Professor, I am working on a final project with one co-author. Can we submit the same short version paper for this HW3?

**Comment by seominjoon** [comment, marked as answer]
Yes, that's fine (if you are both first co-author).

**Comment by seominjoon** [comment]
But make sure to put both names on the paper and indicate that both are first co-authors.
